frontend/src/metabase/public/components/widgets/DisplayOptionsPane.jsx

```diff
@@ -84,7 +84,7 @@ const DisplayOptionsPane = ({
               value={displayOptions.font}
               options={[
                 {
-                  name: t`Use global font`,
+                  name: t`Use default`,
```

Comment:
I guess we're changing this to `Use instance font` now https://metaboat.slack.com/archives/C03E8MQJZBM/p1657127660714979?thread_ts=1657026504.689029&cid=C03E8MQJZBM.

Also, it'd be great if you include some context next time. I checked the Epic (GH) and [the product doc](https://www.notion.so/metabase/Let-whitelabeling-users-set-a-font-family-2813c8fa4d0a40b9b503a66f7a218ce3), but couldn't find anything mentioning this change. So I tried my luck on Slack and found it.